Gemfile.lock

```diff
@@ -291,6 +291,8 @@ GEM
     nokogiri (1.13.1)
       mini_portile2 (~> 2.7.0)
       racc (~> 1.4)
+    nokogiri (1.13.1-x86_64-linux)
```

Comment:
This needs to be platform agnostic
```suggestion
    nokogiri (1.13.1)
```

Please install bundler `2.1.4` and re-bundle
```
gem install bundler -v2.1.4
bundle _2.1.4_ install
```

Reply:
Actually just removing the nokogiri changes here would be enough only payloads gems needed to change.

Reply:
that's actually how they got added for me by using 2.1.4, I had to switch back to 2.2.22 and just manually revert the bundled with line, I'm not sure why that's happened